docs/upgrading/remix.md

```diff
@@ -150,7 +281,7 @@ However, the short-term solution to ease the upgrade is to use TypeScript's [mod
 
 You can do this with the following code in your `vite.config.ts`:
```

Comment:
```suggestion
You can do this with the following code in `react-router.config.ts`:
```

Reply:
We need to make sure that whatever file is used for `interface AppLoadContext` is included in your `tsconfig.json`'s `include`. I'm guessing most people will have `**/*.ts` which will cover this, but might be worth mentioning.

Reply:
Alright, added a comment calling this out, thanks @pcattori